doc/examples/tutorial/example10.json

```diff
@@ -0,0 +1,42 @@
+{
+	/*
+	 * Use case which creates a thread with 3 phases. In each phase the thread
+	 * runs for 2ms and then sleeps for 8ms. In phase0 the thread runs in
+	 * taskgroup /tg1/tg11. Phase1 does not specify a taskgroup so the thread
+	 * will continue to run in taskgroup /tg1/tg11. In phase2 the thread will
+	 * run in taskgroup / (the root taskgroup). The use case runs for 2secs or
+	 * until it is stopped with Ctrl+C.
+	 */
+	"tasks" : {
+		"thread0" : {
+			"loop" : -1,
+			"phases" : {
+				"phase0" : {
+					"run" : 20000,
+					"sleep" : 80000,
+					"taskgroup" : "/tg1/tg11"
+				},
+				"phase1" : {
+					"run" : 20000,
+					"sleep" : 80000
+				},
+				"phase2" : {
+					"run" : 20000,
+					"sleep" : 80000,
+					"taskgroup" : "/"
+				}
+			}
+		}
+	},
+	"global" : {
+		"duration" : 2,
+		"calibration" : "CPU0",
+		"default_policy" : "SCHED_OTHER",
+		"pi_enabled" : false,
+		"lock_pages" : false,
+		"logdir" : "./",
+		"log_basename" : "rt-app1",
+		"ftrace" : true,
```

Comment:
I'd set to false both ftrace and gnuplot.

Reply:
I did look what the other example files are setting this:

deggeman-mac:/opt/git/tools/rt-app:work/scheduler-tools/taskgroups_sched_other_16$ find doc/examples/tutorial/example* | xargs grep ftrace
doc/examples/tutorial/example10.json:           "ftrace" : true,
doc/examples/tutorial/example1.json:            "ftrace" : true,
doc/examples/tutorial/example2.json:            "ftrace" : true,
doc/examples/tutorial/example6.json:            "ftrace" : true,
doc/examples/tutorial/example7.json:            "ftrace" : true,
doc/examples/tutorial/example8.json:            "ftrace" : true,
doc/examples/tutorial/example9.json:            "ftrace" : true,
deggeman-mac:/opt/git/tools/rt-app:work/scheduler-tools/taskgroups_sched_other_16$ find doc/examples/tutorial/example* | xargs grep gnuplot
doc/examples/tutorial/example10.json:           "gnuplot" : true
doc/examples/tutorial/example1.json:            "gnuplot" : true,
doc/examples/tutorial/example2.json:            "gnuplot" : true
doc/examples/tutorial/example6.json:            "gnuplot" : true,
doc/examples/tutorial/example7.json:            "gnuplot" : false,
doc/examples/tutorial/example8.json:            "gnuplot" : true
doc/examples/tutorial/example9.json:            "gnuplot" : true

So except example7.json for gnuplot it's always set to true. Should we handle this with an extra patch for all these example files?

Reply:
Eh.. no, don't bother.. I'll do it eventually. :-)